### ISSUE TITLE
Stop sending data to Kinesis

### ISSUE THREAD

**Issue by davidgasquez** [bufferapp/buda-events-collector#28]
With the Redshift deprecation taking place, we can stop sending some BUDA metrics to AWS. 

These are the current producers.

- [ ] funnel_events
- [ ] `funnels`
- [ ] `subscription_created`
- [ ] `subscription_period_updated`
- [ ] `subscription_cancelled`
- [ ] `visits`
- [ ] `signups`
- [ ] `signins`
- [ ] `actions_taken`

**Comment by davidgasquez** [comment]
I've checked all except `actions_taken`. @michael-erasmus, would you mind quickly checking if that looks ok?

**Comment by michael-erasmus** [comment]
Yes, that looks good to me @davidgasquez! Curious, are we thinking of just removing the code to send to Kinesis?

**Comment by davidgasquez** [comment]
Good one! I went for the AWS side first because I thought that we might need to look at historical data for a certain dataset and might be useful to can keep these datasets flowing into BQ!

We can start only with AWS that and then decide. Go all in and fully stop the service or make it do nothing. :smiley:
